santa/src/before_level.rs

```diff
@@ -249,10 +250,12 @@ fn make_vehicle_panel(ctx: &mut EventCtx, app: &App) -> Panel {
                     .padding(5)
                     .outline(2.0, Color::WHITE)
             } else {
-                let hitbox = batch.get_bounds().get_rectangle();
                 let normal = batch.clone().color(RewriteColor::MakeGrayscale);
                 let hovered = batch;
-                Btn::custom(normal, hovered, hitbox, None).build(ctx, name, None)
+                ButtonBuilder::new()
+                    .custom_batch(normal, ControlState::Default)
```

Comment:
the new `custom_batch` method can always be used in place of the old Btn::custom, but where possible I preferred to move things to using regular builder methods.